sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDBFileManager.scala

```diff
@@ -204,6 +210,9 @@ class RocksDBFileManager(
       case _ =>
         throw QueryExecutionErrors.invalidChangeLogReaderVersion(changelogVersion)
     }
+    if (checkpointUniqueId.isDefined) {
+      changelogReader.lineage
```

Comment:
The way it is written is confusing. Can you improve?

Reply:
Resolved in other PR